### PR TITLE
US635012: OpenText rebrand

### PR DESCRIPTION
**Pull request by michael-bryson**
Ticket: https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=635012

### CODE REVIEW

File: src/main/resources/opentext-swagger-ui.css [hunk 1]
```diff
@@ -23,7 +22,7 @@
 }
 
 .swagger-ui .topbar a {
-    background-image: url(mf_logo_white.svg);
+    background-image: url(VoltageByOpentextSmallWhite.svg);
```
Comment (dermot-hardy) [inline src/main/resources/opentext-swagger-ui.css]: This project is shared across departments.  Updating it to OpenText probably makes sense but not updating it to Voltage.

Comment (michael-bryson) [reply src/main/resources/opentext-swagger-ui.css]: Agreed, this is just a draft and used an image I had handy to test it. Once its working properly I will update to OpenText image instead.

**Comment by michael-bryson** [comment]
FYI, this has been tested here: https://github.com/CAFAudit/audit-service/pull/165
Deployed dev build on david: http://david-int01.swinfra.net:9600/caf-audit-service-ui/#/

**Comment by andyreidz** [comment]
@michael-bryson was there any discussion about the readme.md
"Micro Focus branded version"

**Comment by michael-bryson** [comment]
> @michael-bryson was there any discussion about the readme.md "Micro Focus branded version"

Just forgot to update the readme, done now.

**Review by dermot-hardy — APPROVED** [review]

**Comment by dermot-hardy** [comment]
Looking at this again I would consider the change in the name of the config file (i.e. from `microfocus-config.js` to `opentext-config.js`) to be a breaking change, and I also think that the name of the NPM package needs to be updated.

**Comment by michael-bryson** [comment]
> Looking at this again I would consider the change in the name of the config file (i.e. from `microfocus-config.js` to `opentext-config.js`) to be a breaking change, and I also think that the name of the NPM package needs to be updated.

See https://github.com/MicroFocus/swagger-ui/pull/14

**Pull request merged**